src/urdf_parser_py/urdf.py

```diff
@@ -533,6 +533,20 @@ def get_root(self):
         assert root is not None, "No roots detected, invalid URDF."
         return root
 
+    def post_read_xml(self):
+        if self.version is None:
+            self.version = "1.0"
+
+        split = self.version.split(".")
+        if len(split) != 2:
+            raise Exception("The version attribute should be in the form 'x.y'")
```

Comment:
Recommend a custom exception, but `ValueError` would fit too. Same below.

Reply:
Yeah, I went back and forth on making a custom exception.  I think I'll go with "ValueError", as that seems to make the most sense to me.